go.mod

```diff
@@ -3,20 +3,18 @@ module github.com/solo-io/anyvendor
 go 1.16
 
 require (
-	github.com/envoyproxy/protoc-gen-validate v0.6.1
+	github.com/envoyproxy/protoc-gen-validate v0.6.7
 	github.com/go-git/go-git/v5 v5.3.0
 	github.com/golang/mock v1.6.0
 	github.com/golang/protobuf v1.5.2
-	github.com/iancoleman/strcase v0.1.3 // indirect
-	github.com/lyft/protoc-gen-star v0.5.3 // indirect
+	github.com/lyft/protoc-gen-star v0.6.1 // indirect
 	github.com/mattn/go-zglob v0.0.3
 	github.com/onsi/ginkgo v1.11.0
 	github.com/onsi/gomega v1.8.1
 	github.com/rotisserie/eris v0.1.1
-	github.com/spf13/afero v1.6.0
-	golang.org/x/lint v0.0.0-20210508222113-6edffad5e616 // indirect
-	golang.org/x/net v0.0.0-20210614182718-04defd469f4e // indirect
-	golang.org/x/sys v0.0.0-20210630005230-0f9fa26af87c // indirect
-	golang.org/x/tools v0.1.4 // indirect
-	google.golang.org/protobuf v1.27.1 // indirect
+	github.com/spf13/afero v1.9.2
+	golang.org/x/net v0.0.0-20220909164309-bea034e7d591 // indirect
+	golang.org/x/sys v0.0.0-20220913120320-3275c407cedc // indirect
+	golang.org/x/tools v0.1.12 // indirect
+	google.golang.org/protobuf v1.28.1 // indirect
```

Comment:
None of these changes are necessary, and seem compelled mainly by the codegen check in CI, which calls
https://github.com/solo-io/anyvendor/blob/cd2bd5f65d0b8b69ec81205d5b872b136de02ab0/Makefile#L25-L34

.. which to me seems a bit aggressive at compelling upgrades?  At the very least, this is the first I've seen that sort of thing.  Could you maybe look into why it's there?   I'm worried this will be a problem for GM and GE if this is a fundamental dependency.

Reply:
The update-deps command should be a prerequisite for running make generated-code , but should not have any dependencies that are pulled in to other repos.